hrms/hr/report/shift_attendance/shift_attendance.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2023, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+from frappe.utils import flt
+
+
+def execute(filters=None):
+	columns = get_columns()
+	data = get_data(filters)
+	return columns, data
```

Comment:
Let's also add a chart & a report summary to enhance the UX and summarize the data. These will get affected by filters (so if you filter by employee, it should show the employee's summary)

https://frappeframework.com/docs/user/en/desk/reports/script-report#report-summary

The report summary can show the number of:
- present
- absent
- half day records
- late entries
- early exits

eg:
![image](https://github.com/frappe/hrms/assets/24353136/1fb556b7-9cfc-45d9-9605-efdda2d99a33)

Add a chart for Shift Type breakup